server/repository/src/migrations/v2_03_00/return_types_rename.rs

```diff
@@ -0,0 +1,24 @@
+use crate::{migrations::*, StorageConnection};
+
+pub fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    if cfg!(feature = "postgres") {
+        sql!(
+            connection,
+            r#"
+                 ALTER TYPE context_type RENAME VALUE 'INBOUND_RETURN' TO 'CUSTOMER_RETURN';
+                 ALTER TYPE context_type RENAME VALUE 'OUTBOUND_RETURN' TO 'SUPPLIER_RETURN';
+                 ALTER TYPE invoice_type RENAME VALUE 'INBOUND_RETURN' TO 'CUSTOMER_RETURN';
+                 ALTER TYPE invoice_type RENAME VALUE 'OUTBOUND_RETURN' TO 'SUPPLIER_RETURN';
+                 ALTER TYPE number_type RENAME VALUE 'INBOUND_RETURN' TO 'CUSTOMER_RETURN';
+                 ALTER TYPE number_type RENAME VALUE 'OUTBOUND_RETURN' TO 'SUPPLIER_RETURN';
```

Comment:
I think these need to be add rather than alter? Unless we have another pattern for migrating existing data? Getting this error when I re-initialise:

```unknown variant `OUTBOUND_RETURN`, expected one of `OUTBOUND_SHIPMENT`, `INBOUND_SHIPMENT`, `PRESCRIPTION`, `INVENTORY_ADDITION`, `INVENTORY_REDUCTION`, `REPACK`, `SUPPLIER_RETURN`, `CUSTOMER_RETURN` "2C0ED7C4C9C04B25BAC532C26B2758D0" "transact"```

Reply:
Oof yah and then the transfer processors are like hey the transferred return hasn't been created yet and its creating new customer returns 👀

Reply:
This is right for postgres, forgot to rename sql XD

Reply:
The rename works for migrating 2.2 to 2.3, but when you reinitialise, there are `INBOUND_RETURN` and `OUTBOUND_RETURN` transact records in mSupply and translators don't know what to do... I think the migrations here are fine, but need to add a serde rename to the translator?

Reply:
When I re-initialise the INBOUND_RETURN and OUTBOUND_RETURN transactions are renamed to CUSTOMER_RETURN and SUPPLIER_RETURN? 🤔

Reply:
Postgres?

Reply:
postgres works fine on re-initialisation too...

Reply:
[65f4143](https://github.com/msupply-foundation/open-msupply/pull/4610/commits/65f4143cbcd90bda707613e179be17fe379b1eb1)